src/go/rpk/pkg/cli/cmd/debug/bundle_linux.go

```diff
@@ -55,17 +55,36 @@ func executeBundle(
 	logsSince, logsUntil string,
 	logsLimitBytes int,
 	timeout time.Duration,
+	pathname string,
 ) error {
+	fmt.Println("Creating bundle file...")
 	mode := os.FileMode(0o755)
-	timestamp := time.Now().Unix()
-	filename := fmt.Sprintf("%d-bundle.zip", timestamp)
 	f, err := fs.OpenFile(
-		filename,
+		pathname,
 		os.O_CREATE|os.O_WRONLY,
 		mode,
 	)
 	if err != nil {
-		return fmt.Errorf("couldn't create bundle file: %w", err)
+		if !errors.Is(err, os.ErrPermission) {
```

Comment:
Why fallback to $HOME? I'm not sure of any other tool that does this, so it feels a bit odd from a UX perspective. If I requested a file to be written to /foo/bar.zip, and I cannot, I wouldn't expect it to fallback somewhere else?

Reply:
Agree, my bad. My intention is:

**Scenario:**  you set the output-file flag:
- Write to that filepath
- If you don't have write permissions: fail.

**Scenario:**  you don't set the flag:
- Write to `pwd` (to keep backompat)
- If you don't have write permissions: default to $HOME

Reply:
Changed, lmk what do you think